src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs

```diff
@@ -80,21 +69,50 @@ protected override void ProcessRecord()
                     if (!string.IsNullOrEmpty(html))
                     {
                         string tmpFilePath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString() + ".html");
-                        using (var writer = new StreamWriter(new FileStream(tmpFilePath, FileMode.Create, FileAccess.Write, FileShare.Write)))
+
+                        try
                         {
-                            writer.Write(html);
+                            using (var writer = new StreamWriter(new FileStream(tmpFilePath, FileMode.Create, FileAccess.Write, FileShare.Write)))
```

Comment:
The same - we could use one `try` block instead of `try-using`.

Reply:
I prefer `using` since there are multiple catch blocks and disposing in catch will be far too disjoint from where StreamWriter / FileStream are used.